javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
Oh, you mean losing other properties than `message`. That is a pro for Inspector, as we want to see data like:
```js
obj = { err1: Error('Error1'),  err2: Error('Error2') }
```

Having a lot of information makes it difficult for visual inspection. Even though for other cases it could be useful to have stack trace, we still wouldn't be able to restore it on parsing (on eval).